.github/workflows/test_install.yml

```diff
@@ -85,6 +85,18 @@ jobs:
       with:
         python-version: "${{ inputs.python_version }}"
 
+    - name: Install Linux Dependencies
+      if: matrix.os == 'ubuntu-latest'
```

Comment:
There's no strategy matrix on this job. I think the left side here needs to be `${{ needs.audit_inputs.outputs.runs-on }}`.